src/main/java/net/comments/objects/BCommentTable.java

```diff
@@ -0,0 +1,39 @@
+package net.comments.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BCommentTable implements CommentTable {
+    private final WebDriver driver;
+    private List<Comment> commentsList;
```

Comment:
Please define this variable within `getComments` method as it is used only there.